usr/bin/irdata_parse_uniprot.sh

```diff
@@ -62,30 +62,31 @@ for FILENAME in $FILENAMES; do
         # Unpack any gzip archives since the slicing of these files is not
         # efficient if done repeatedly.
 
-        if [ "$FILETYPE" = 'dat.gz' ]; then
-            UNPACKED_FILENAME=${FILENAME%.gz}
+        ## Leave file compressed
+        # if [ "$FILETYPE" = 'dat.gz' ]; then
+        #     UNPACKED_FILENAME=${FILENAME%.gz}
```

Comment:
Can the commented code in this file be deleted?

Reply:
I have a follow up change ready dealing specifically with the more efficient parsing of compressed flat files. The comment will be deleted there.